code/controllers/subsystem/silo.dm

```diff
@@ -29,6 +34,8 @@ SUBSYSTEM_DEF(silo)
 	//We scale the rate based on the current ratio of humans to xenos
 	current_larva_spawn_rate *= clamp(round((active_humans / active_xenos) / (LARVA_POINTS_REGULAR / xeno_job.job_points_needed), 0.01), 0.5, 1)
 
+	current_larva_spawn_rate *= LERP(1, XENO_SPAWN_TIME_SCALING_MAX_AMOUNT, min((world.time - SSticker?.round_start_time) / XENO_SPAWN_TIME_SCALING_MAX_TIME, 1))
```

Comment:
regardless of my aforementioned opinion on bigger-picture problems and onto the actual code after looking at this again:
would this indicate that the scaling begins from the start of the round? would there be a way to have it start from 12:18 (minimum deployment time, usually) instead so that there aren't 15 minutes that count into the scaling factor that marines aren't realistically utilizing while groundside?

Reply:
Yeah would probably be good if this ticked only after shutters open.